commitizen/cz/base.py

```diff
@@ -1,13 +1,29 @@
-from typing import Optional
+from typing import Optional, List, Tuple
 from abc import ABCMeta, abstractmethod
 
+from prompt_toolkit.styles import merge_styles, Style
+
 
 class BaseCommitizen(metaclass=ABCMeta):
     bump_pattern: Optional[str] = None
     bump_map: Optional[dict] = None
+    DEFAULT_STYLE_CONFIG: List[Tuple[str, str]] = [
```

Comment:
What do you think of making this lower case to be consistent with the other attributes?

Reply:
I agree with you. I've updated it. Thanks!